save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/Forum.kt

```diff
@@ -50,71 +51,71 @@ val newCommentWindow: FC<NewCommentWindowProps> = FC { props ->
+        className = ClassName("shadow row no-gutters mx-auto input-group border border-secondary")
         renderLeftColumn(
-            user.avatar,
-            user.name,
-            user.rating,
+            props.currentUserInfo?.avatar,
+            props.currentUserInfo?.name,
+            props.currentUserInfo?.rating,
+            "#ceedc7"
         )
         div {
-            className = ClassName("card col-10 input-group needs-validation")
+            className = ClassName("col shadow")
             textarea {
-                className = ClassName("form-control")
+                className = ClassName("form-control p-3 border-0")
                 style = jso {
                     width = "100%".unsafeCast<Width>()
                     height = "100%".unsafeCast<Height>()
                 }
                 onChange = { event ->
                     setComment { it.copy(message = event.target.value) }
                 }
+                value = comment.message
                 ariaDescribedBy = "${InputTypes.COMMENT.name}Span"
                 rows = 5
                 id = InputTypes.COMMENT.name
                 required = true
+                placeholder = "Write a comment"
             }
         }
     }
     div {
         className = ClassName("d-flex justify-content-end")
-        buttonBuilder(label = "Comment") {
+        buttonBuilder(faPaperPlane, isDisabled = comment.message.isBlank(), classes = "mt-3") {
             enrollRequest()
         }
     }
 }
 
 /**
- * @return a function component
+ * [FC] for comment displaying
  */
-@Suppress(
-    "GENERIC_VARIABLE_WRONG_DECLARATION",
-    "MAGIC_NUMBER",
-)
+@Suppress("GENERIC_VARIABLE_WRONG_DECLARATION", "MAGIC_NUMBER")
 val commentWindow: FC<CommentWindowProps> = FC { props ->
-
-    val columnCard = cardComponent(isBordered = false, hasBg = true, isNoPadding = false, isPaddingBottomNull = true, isFilling = true)
-
     div {
-        className = ClassName("row no-gutters mx-auto border-secondary")
+        className = ClassName("shadow input-group row no-gutters mx-auto border-secondary")
         renderLeftColumn(
             props.comment.userAvatar,
             props.comment.userName,
             props.comment.userRating,
         )
         div {
-            className = ClassName("card col-10 text-left border-secondary")
+            className = ClassName("card col-10 text-left border-0")
             val comment = props.comment
             div {
-                className = ClassName("col")
-                style = jso {
-                    background = "#F1F1F1".unsafeCast<Background>()
+                className = ClassName("flex-wrap")
+                style = jso { background = "#F1F1F1".unsafeCast<Background>() }
+                span {
+                    className = ClassName("ml-1")
+                    +(comment.createDate?.toUnixCalendarFormat(TimeZone.currentSystemDefault()) ?: "Unknown")
```

Comment:
why date is nullable?

Reply:
Have no idea
I have just refactored it

Reply:
The date is filled in on the backend. Therefore, in DTO, when creating a new comment, we pass null.

Reply:
It was my mistake, fixed it in https://github.com/saveourtool/save-cloud/pull/2260/commits/6bd9fbed297def21058a4860c4197fb708912bc2